website/docs/r/log_search.html.markdown

```diff
@@ -73,7 +73,8 @@ The following arguments are supported:
 - `description` - (Optional) Description of the search.
 - `parent_id` - (Required) The identifier of the folder to create the log search in.
 - `query_string` - (Required) Log query to perform.
-- `query_parameters` - (Optional) TODO Find a good description.
+- `query_parameters` - (Optional) Values for search template used in the search query. 
```

Comment:
What is an example of this in terraform? I assume this is a complex type. Does this nest blocks, is it a list? As a user I don't see examples of this in the documentation, so it would be helpful to know the schema of this argument.